packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -109,11 +109,20 @@ const CARDANO_FEE_INFO: FeeInfoWithLevels = {
     dustLimit: -1, // unknown/unused
 };
 
+const SOLANA_FEE_INFO: FeeInfoWithLevels = {
+    blockTime: -1, // unknown
+    defaultFees: [{ label: 'normal', feePerUnit: '5000', blocks: -1 }],
+    minFee: 5000,
+    maxFee: 100000000, // arbitrary
```

Comment:
Solana fees can essentially be arbitrary. How should we handle this?
Right now, the highest fees we can produce are `2044280` from transactions creating associated token accounts, [example](https://explorer.solana.com/address/ETxHeBBcuw9Yu4dGuP3oXrD12V5RECvmi8ogQ9PkjyVF?cluster=devnet)

Reply:
🤷‍♂️